server/controllers/users.js

```diff
@@ -171,3 +171,19 @@ export function getUserCampaigns(req, res, next) {
     })
     .catch(err => console.log('could not retrieve user\'s campaigns:', err));
 }
+
+export function getUserCampaignAssociation(req, res) {
+  const { id, campaign_id } = req.params;
+  const params = { id, campaign_id };
+  return usersService.getUserCampaignAssociation(params)
+    .then((campaign) => {
+      if (campaign !== null) {
+        res.sendStatus(204);
+      } else {
+        res.sendStatus(404);
+      }
+    })
+    .catch((err) => {
+      res.status(404).json({ message: `This association does not exist.: ${err}` });
```

Comment:
Note: this error is unlikely to be the cause of association not existing (your else block above will catch it). This is likely going to be a higher level DB/server error. You may want to update the error message to something more genetic.

Reply:
Updated!